docs/siem/detections/api/lists/api-update-list-item.asciidoc

```diff
@@ -0,0 +1,79 @@
+[[lists-api-update-item]]
+=== Update list item
+
+Updates an existing list item.
+
+You can use `PUT` or `PATCH` methods to update list items, where:
+
+* `PUT` replaces the original items and deletes fields that are not
+specified.
+* `PATCH` updates the specified fields.
+
+==== Request URL
+
+`PUT <kibana host>:<port>/api/lists/items`
+
+`PATCH <kibana host>:<port>/api/lists/items`
+
+==== Request body
+
+A JSON object with:
+
+* The `id` of the list item you want to update.
+* The fields you want to modify.
+
+IMPORTANT: If you call `PUT` to update a rule, all unspecified fields are
+deleted. You cannot modify the `list_id` and `id` fields.
+
+For `PATCH` calls, any of the fields can be modified, whereas for `PUT` calls,
+some fields are required. 
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required (`PUT` calls)
+
+|`meta` |Object |Placeholder for metadata about the list item. |No
+|`value` |String |The value to be excluded. |Yes
```

Comment:
Same comment as above that values are agnostic in a sense in that later features will use them for things other than exclusions.